i18n/en/docusaurus-plugin-content-docs/current/index.mdx

```diff
@@ -5,3 +5,122 @@ sidebar_position: 1
 # Overview
 
 **Testplane** — a tool for automating testing of user scenarios in web interfaces.
+
+With Testplane, you can test the functionality of your site or project and ensure that your layout consistently matches what's expected under any circumstances. At your disposal, you have the full range of [commands][commands] provided by [WebDriverIO v8][webdriverio-api] and [all the capabilities of the Chrome DevTools protocol][how-to-use-cdp]. Additionally, you have the familiar [Mocha][mocha] syntax.
```

Comment:
> `With Testplane,`

Are you sure here needs to be a comma? They don't use a lot of those in english

> `that`

This one either

Reply:
Researched this comma question for a bit, it seems like comma is optional in this case, but not a mistake. It is "introductory prepositional phrase" and comma may be used here to improve readability. Keeping it for now.